Patterns/Dutch/Dutch-DateTime.yaml

```
@@ -1364,4 +1366,16 @@ YearToDateTerms: !list
     - jaar tot heden
     - jaar tot op heden
     - vanaf vorig jaareinde
+DayTypeRegex: !simpleRegex
+  def: ^(dag(elijks)?)$
+WeekTypeRegex: !simpleRegex
+  def: ^(wekelijks|week)$
+BiWeekTypeRegex: !simpleRegex
+  def: ^(tweewekelijks)$
+MonthTypeRegex: !simpleRegex
+  def: ^(maand(elijks)?)$
+QuarterTypeRegex: !simpleRegex
+  def: ^(kwartaal|driemaandelijkse)$
```

```suggestion
  def: ^(kwartaal|driemaandelijks)$
```

Form would be `driemaandelijks` instead of `driemaandelijkse`. (Same form as DayTypeRegex, WeekTypeRegex etc.).

---

In the end, best way forward would be to add the optional 'e' suffix to all forms (`wekelijks(e)?`), since this is a common form used in a sentence. But not sure if this is something that should be part of this PR.

Example: wekelijkse boodschappen. (weekly groceries)

---

added optional 'e' suffix to these patterns.

---

@tellarin do these new additions be covered by specs as well? If so; hereby some example sentences that could make sense:

` Lees de beste artikelen op het gebied van Contoso via onze wekelijkse nieuwsbrief!`
(Read the best articles on the field of Contoso via our weekly newsletter)

`Laten we een driemaandelijkse vergadering houden. `
(Let's have a quarterly meeting.)

---

@LionbridgeCSII, can you add these cases suggested by @iMicknl?

---

added